crowd.cfg.tmpl

```diff
@@ -2,9 +2,10 @@
 
 <application-configuration>
   <setupStep>complete</setupStep>
-  <setupType>install.xml</setupType>
-  <buildNumber>665</buildNumber>
+  <setupType>install.new</setupType>
```

Comment:
I'm a little surprised this changed… IIRC the XML setup type was used when it imported the config from an XML file. Maybe this is just different in Crowd 3 but I thought it was worth double-checking.

Reply:
When configuring the new 3.3 container, I copy/pasted the crowd.cfg that got generated at the end of the install - since I didn't restore from an xml backup, I assume that's why it switched.

I'm not certain if it needs to be changed if the old version was also `install.xml` - my 2.12 test was a fresh install as well, so I went from `install.new` to `install.new`.  I could test with install.xml and see if anything breaks.

Reply:
Oh, I see, maybe it indicates restoring from an XML backup and not loading the XML config file. If it's starting up fully configured, it's probably fine.